### ISSUE TITLE
Pythonnet 3.0 Enums with integer

### ISSUE THREAD

**Issue by hamzaselmancelik** [pythonnet/pythonnet#1971]
### Environment

-   Pythonnet version: 3.0
-   Python version: 3.8
-   Operating System: Windows

### Details

-   I was using Pythonnet 2.5.2, and I have a C# Languages Enum. I have a HowManyLanguagesDoYouSpeak method on C#:
        public int HowManyLanguagesDoYouSpeak(Languages languages)
        {
            return Enum.GetValues(typeof(Languages)).Cast<Languages>()
                .Where(lang => Languages.NoLanguage != lang)
                .Where(lang => (languages & lang).Equals(lang)).Count();
        }

    I was calling this method python with pythonnet with:
    result = self.cs_object.HowManyLanguagesDoYouSpeak(5)
    
    It used to work just fine with pythonnet 2.5.2. Now that I updated my pythonnet into 3.0. It does not work anymore. I get an error

     ![image](https://user-images.githubusercontent.com/46783026/195132807-8fca30a7-2382-48c6-868b-0a9d54d2a733.png)

     I think this needs to be solved on pythonnet 3.0

**Comment by lostmsu** [comment]
No, this is by design. You need to pass `Languages.NoLanguage` or one of other values from `Languages` instead of the number.
